### PR TITLE
Add updated package version

### PR DESCRIPTION
**Pull request by sbrunner**
GSGMF-1658

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
...
+* svgo from 1.2.2 to 1.3.2
+* ts-node from 8.1.0 to 8.8.2
+* tsconfig-paths from 3.8.0 to 3.9.0
+* webpack from 4.30.0 to 4.42.1
+* webpack-dev-server from 3.3.1 to 3.11.0
+
+Application build
+.................
+
+In the new version the build of the allication becomes more Docker fiendly.
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:258]:
```suggestion
In the new version the build of the application becomes more Docker friendly.
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 2]
```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
...
+We didn't anymore havbe a build container that build files in the local filesystem, new everything is done
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:259]:
```suggestion
We didn't anymore have a build container that builds files in the local filesystem, now everything is done
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 3]
```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
...
+in Docker. Tho only thing that runs out of docker is the `build` script (that needs Python 3.x) who run some
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:260]:
```suggestion
in Docker. The only thing that runs out of docker is the `build` script (that needs Python 3.x) which runs some
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 4]
```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
...
+`docker build` command, and the `docker-compose`.
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:261]:
```suggestion
`docker build` and `docker-compose` commands.
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 5]
```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
 Version 2.5.0
 =============
 
+Information to know before starting the upgrade
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:7]:
```suggestion
Informations to know before starting the upgrade
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 6]
```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
...
+-----------------------------------------------
+
+Main version update
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:10]:
```suggestion
Main versions updates
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 7]
```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
...
+...................
+
+Ubuntu is updated from 18.04.2 to 18.04.5
+Python is updated from 3.6.7 to 3.7.5
+Node is updated from 10.15.3 to 10.24.1
+Npm is updated from 6.4.1 to 6.14.12
+
+Python package update
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:18]:
```suggestion
Python packages updates
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 8]
```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
...
+* redis from 3.2.1 to 3.4.1
+* requests from 2.21.0 to 2.23.0
+* Shapely from 1.6.4.post2 to 1.7.0
+* simplejson from 3.16.0 to 3.17.2
+* six from 1.11.0 to 1.14.0
+* urllib3 from 1.23 to 1.25.9
+* waitress from 1.2.1 to 1.4.3
+* zope.sqlalchemy from 1.1 to 1.3
+
+Npm package update
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:178]:
```suggestion
Npm packages updates
```

**Review by arnaud-morvan — APPROVED** [review]

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl (hunk 5 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:7]: As I know Information has never 's', not?

Comment (arnaud-morvan) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl:7]: Seems you're right on this one (as says google translate).

**Pull request merged**